<ui version="4.0" >
 <class>SnapshotForm</class>
 <widget class="QWidget" name="SnapshotForm" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>344</width>
    <height>189</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_4" >
   <item>
    <layout class="QVBoxLayout" name="verticalLayout" >
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout" >
       <item>
        <widget class="QLabel" name="textLabel1" >
         <property name="text" >
          <string>File format for Shift-S snapshots</string>
         </property>
         <property name="wordWrap" >
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="formatComboBox" >
         <property name="toolTip" >
          <string>Select file format for snapshots done with Shift-S</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="otherFormatsLabel" >
       <property name="text" >
        <string>(Ctrl-S gives TIFF, Ctrl-Shift-S gives PNG)</string>
       </property>
       <property name="alignment" >
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2" >
       <item>
        <widget class="QLabel" name="textLabel2" >
         <property name="toolTip" >
          <string>Factor determining compression/quality of JPEG images</string>
         </property>
         <property name="text" >
          <string>Quality factor for JPEG images (%)</string>
         </property>
         <property name="wordWrap" >
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="qualitySpinBox" >
         <property name="toolTip" >
          <string>Factor determining compression/quality of JPEG images (-1 for default)</string>
         </property>
         <property name="minimum" >
          <number>-1</number>
         </property>
         <property name="maximum" >
          <number>100</number>
         </property>
         <property name="value" >
          <number>80</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3" >
       <item>
        <widget class="QLabel" name="label_2" >
         <property name="text" >
          <string>Dots per inch to specify in output file:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="dpiSpinBox" >
         <property name="toolTip" >
          <string>Resolution value (DPI) to set in file (0 for none)</string>
         </property>
         <property name="specialValueText" >
          <string>None</string>
         </property>
         <property name="maximum" >
          <number>10000</number>
         </property>
         <property name="singleStep" >
          <number>50</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QCheckBox" name="scaleDPIcheckBox" >
       <property name="toolTip" >
        <string>Scale resolution value up so that montaged snapshot will display the same size as regular snapshot</string>
       </property>
       <property name="text" >
        <string>Scale DPI by zoom factor for montage snapshots</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer" >
       <property name="orientation" >
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="horizontalSpacer" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>13</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
